.pre-commit-config.yaml

```diff
@@ -25,7 +25,7 @@ repos:
     hooks:
       - id: code-format
         name: code-format
-        entry: bash -c './gradlew spotlessCheck || (echo ""; echo "!!Automatically running spotlessApply!!"; ./gradlew spotlessApply)'
+        entry: bash -c './gradlew spotlessCheck || (echo ""; echo "!!Automatically running spotlessApply!!"; ./gradlew spotlessApply && git status)'
```

Comment:
no committing. I decided to not add the unstaged files, this gives the user the flexibility to add the files as they wish. The only thing I added was showing the unstaged files, in case the linter made any changes.